server/gitrest/packages/gitrest-base/.eslintrc.cjs

```diff
@@ -4,12 +4,18 @@
  */
 
 module.exports = {
-	extends: [require.resolve("@fluidframework/eslint-config-fluid/minimal"), "prettier"],
+	extends: [
+		require.resolve("@fluidframework/eslint-config-fluid/minimal-deprecated"),
+		"prettier",
+	],
 	parserOptions: {
 		project: ["./tsconfig.json", "./src/test/tsconfig.json"],
 	},
 	rules: {
-		"@typescript-eslint/strict-boolean-expressions": "off", // requires strictNullChecks=true in tsconfig
 		"import/no-nodejs-modules": "off",
+
+		// TODO: enable strict null checks in tsconfig and remove these overrides
+		"@typescript-eslint/prefer-nullish-coalescing": "off",
```

Comment:
Had to add this rule because of changes in the minimal eslint config from 2.0 to 4.0?

Reply:
Yeah. I had to add it to all of the server packages that didn't have `strictNullChecks` enabled (which was all but I think 2).

Reply:
There was at least one package with `strictNullChecks` that I also disabled the rule for (with a different comment) due to the sheer number of violations. I wanted to minimize the number of code changes I made to only the most trivial.